firebase-appdistribution/test-app/test-app.gradle

```diff
@@ -73,6 +84,7 @@ dependencies {
 
     // All variants use the API
     implementation project(':firebase-appdistribution-api:ktx')
+//    implementation 'com.google.firebase:firebase-appdistribution-api-ktx:16.0.0-beta08'
```

Comment:
delete outright instead of adding comment?

Reply:
Did not mean to commit any of this, sorry